modules/plugins/utility/motion/precognition/precognition.nix

```diff
@@ -0,0 +1,78 @@
+{lib, ...}: let
+  inherit (lib.options) mkEnableOption mkOption literalExpression;
+  inherit (lib.types) attrsOf listOf str bool int submodule;
+in {
+  options.vim.utility.motion.precognition = rec {
+    enable = mkEnableOption "precognition.nvim plugin";
+
+    startVisible = mkOption {
+      type = bool;
+      description = "Whether to start 'precognition' automatically.";
+      default = true;
+    };
+
+    showBlankVirtLine = mkOption {
+      type = bool;
+      description = "Whether to show a blank virtual line when no movements are shown.";
+      default = true;
+    };
+
+    highlightColor = mkOption {
+      type = attrsOf str;
+
+      example = literalExpression ''
+        { link = "Comment"; }
+        # or
+        { foreground = "#0000FF"; background = "#000000"; };
+      '';
+      default = {link = "Comment";};
+      description = ''
+        The highlight for the virtual text.
+
+        To find more about this option, please see
+        [the documentation](https://github.com/tris203/precognition.nvim).
+      '';
+    };
+
+    hints = mkOption {
+      default = {};
+      description = ''
+        What motions display and at what priority.";
+
+        To find more about this option, please see
+        [the documentation](https://github.com/tris203/precognition.nvim).
```

Comment:
I don't think we need to link the documentation again for every option: the user should be able to look it up or find it in one of  changelog, or the commit description.

Please remove it here and from any other options that link to precognition.nvim page.

Reply:
While you say that users should be able to find the commit or changelog entry, you also said to clarify my use of "the documentation" prior. Am I to remove _all_ including the example from the last suggestion, or to only leave the first occurrence?

I appreciate the feedback.

Reply:
If there is an option that requires special clarification (e.g. examples, detailed implementation, etc.) to make sense, feel free to include a link to the documentation of the plugin,or the repository, in the description.

If not, please add a link to the plugin page in the changelog and leave it at that. The plugin in question is easy to find, so we can trust the user to find the appropriate documentation, should they get stuck.

Reply:
Resolved in 0b647a209db5acd709506df510e0d9fea26aa7a3.